umake/network/download_center.py

```diff
@@ -34,6 +35,7 @@
 from umake.tools import ChecksumType
 
 logger = logging.getLogger(__name__)
+MAX_WORKERS = cpu_count()
```

Comment:
I don't really like executing code

Also, same comment than in most of part, please do not make any changes unrelated to the current PR, and separate those as well in different commits. (so good for another PR, but that one should be independant as well from the refactoring which should be a second one).

Now, on the change:
I don't like executing code when importing a module (apart from setting up loggers and small things like that). Mind moving that to tools.py? and reference it here?